### ISSUE TITLE
Testing fails on Windows

### ISSUE THREAD

**Issue by SandySingh** [RBerkheimer/harness#11]
When I tried to run harness on windows, it gives an error while testing - 

> FileNotFoundError: [Errno 2] No such file or directory: 'D:\\Project_Harness\\venv\\lib\\site-packages\\harness\\src\\main\\dao\\filesystemstandard'

**Comment by RBerkheimer** [comment]
We will need a user with access to Windows to test.

@SandySingh once you have it set up on a nix VM, can we telecon to work this out? In the meanwhile, I'll try digging out an old machine.

**Comment by SandySingh** [comment]
Yeah sure, but I think my VM will take some time as I am facing some internet connectivity issues.
